docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,267 @@
+---
+title: UID2 Operator - Google Cloud Platform Confidential Space
+sidebar_label: GCP Confidential Space
+pagination_label: UID2 Operator - Google Cloud Platform Confidential Space
+description: Integration information for Google Cloud Platform private operator.
+hide_table_of_contents: false
+sidebar_position: 20
+---
+
+# UID2 Private Operator - Google Cloud Platform Confidential Space
+
+This guide provides information for setting up the UID2 Operator Service in a secure enclave in a [Google Cloud](https://cloud.google.com/docs/overview/) platform [Confidential Space](https://cloud.google.com/confidential-computing#confidential-space).
+
+<!-- It includes the following information:
+
+- [Overview](#overview)
+- [Terminology](#terminology)
+- [Attestation Requirements](#attestation-requirements)
+- [Integration Deployment](#integration-deployment)
+  - [Cloud-init.yaml File](#cloud-inityaml-file)
+  - [Cloud-init Example](#cloud-init-example)
+  - [Create VM Instance](#create-vm-instance)
+- [Production Deployment](#production-deployment)
+- [Upgrading](#upgrading) 
+-->
+
+## Overview
+
+Confidential Space, from Google Cloud Platform, is GCP's latest confidential computing option, with a GA date of March 2023.
+
+The UID2 GCP Confidential Space private operator solution offers a secure enclave environment, known as a <a href="#gl-tee">Trusted Execution Environment (TEE)</a>.
+
+Confidential Space enables the sharing of confidential data in an isolated and secure environment so that privacy, confidentiality, and ownership of the data are preserved.
+
+(**GWH_YS01 do we need to say anything about how this is better than the previous solution?**)
```

Comment:
Previous solution didn't provide a true isolated and immutable environement.
We may not want to explicitly explain those vulnerabilities though.